Micronaut/src/main/kotlin/io/micronaut/turbo/basic/NonExistentController.kt

```diff
@@ -0,0 +1,15 @@
+package io.micronaut.turbo.basic
+
+import io.micronaut.http.HttpStatus
+import io.micronaut.http.MediaType
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.annotation.Status
+
+@Controller("/nonexistent")
+class NonExistentController {
+
+    @Status(HttpStatus.NOT_FOUND)
+    @Get(produces = [MediaType.TEXT_HTML], consumes = [MediaType.TEXT_HTML])
+    fun index() = "Not Found"
```

Comment:
Is this missing the `@View` annotation. We say we are returning HTML wit the content-type but it seems to return just a string.

Reply:
In their app, they just return a 404 and a Sting, so I copied that

https://github.com/hotwired/turbo-native-demo/blob/fbd6cb6e07eb4fe1661d6330a377237d6ce606a8/server.js#L117-L119